apps/remix-ide/src/app/files/fileManager.ts

```diff
@@ -547,10 +544,10 @@ class FileManager extends Plugin {
     })
   }
 
-  _saveAsCopy (path, content) {
+  _saveAsCopy(path, content) {
     const fileProvider = this.fileProviderOf(path)
     if (fileProvider) {
-      helper.createNonClashingNameWithPrefix(path, fileProvider, '', (error, copyName) => {
+      helper.createNonClashingNameAsync(path, this, '', (error, copyName) => {
```

Comment:
`createNonClashingNameAsync` doesn't accept a callback. You need to update the implementation.